cmd/osbuild-dnf-json-tests/main_test.go

```diff
@@ -0,0 +1,46 @@
+// This package contains tests related to dnf-json and rpmmd package.
+package main
+
+import (
+	"fmt"
+	"github.com/osbuild/osbuild-composer/internal/rpmmd"
+	"os/exec"
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/Flaque/filet"
+	"path"
+)
+
+func setUpTemporaryRepository(dir string) (string, error) {
+	cmd := exec.Command("createrepo_c", dir)
+	err := cmd.Start()
+	if err != nil {
+		return "", err
+	}
+	err = cmd.Wait()
+	if err != nil {
+		return "", err
+	}
+	return dir, nil
+}
+
+func TestFetchChecksum(t *testing.T) {
+	defer filet.CleanUp(t)
+	
+	dir := filet.TmpDir(t, "/tmp")
```

Comment:
What was the reason to use this go module?

Reply:
I'm not really sure. I was seeing some errors initially which may not have been related to the temporary directory at all. I will revert this change and try again.